src/Maker/MakeRegistrationForm.php

```diff
@@ -328,6 +339,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                     'email_getter' => $this->emailGetter,
                     'authenticator_class_name' => $this->autoLoginAuthenticator ? Str::getShortClassName($this->autoLoginAuthenticator) : null,
                     'authenticator_full_class_name' => $this->autoLoginAuthenticator,
+                    'uses_authenticator_system' => $this->useSecurity52,
```

Comment:
```suggestion
                    'use_new_authenticator_system' => $this->useNewAuthenticatorSystem,
```
Looks clearer to me, if needed we can add 52 suffix to the variable name